tests/integration/tier0/non-destructive/file-backup/test_file_backup.py

```diff
@@ -13,12 +13,11 @@
     "yum": "/etc/logrotate.d/yum",
     "dnf": "/etc/logrotate.d/dnf",
 }
-BACKUP_DIR = "/tmp/c2r_tests_backup/"
-MODIFIED_FILES_DIR = "/tmp/c2r_tests_backup/modified/"
 
 
 @pytest.fixture
-def config_files_modified(shell):
+@pytest.mark.config_files_modified
```

Comment:
Just a suggestion, maybe we should prefix those markers that are created because of the `backup_directory`. Something like:
```suggestion
@pytest.mark.backupdir_config_files_modified
```

Reply:
Good idea, we can do that in a separate PR